modules/mongodb/mongodb_test.go

```diff
@@ -1,21 +1,22 @@
-package mongodb_test
+package mongodb
```

Comment:
Even tough there could be packages with the tests in the same package than the module, I'd not change this, as this is the preferred way to test the module.

I can open an issue to update those tests that live in the same package as its module.

Reply:
I reverted it to mongodb_test. Out of curiosity, why do you want test to be in a separate package? Is to make sure importers of the package shouldn't use the test methods ?

Reply:
>  Is to make sure importers of the package shouldn't use the test methods ?

Indeed, and also for "feeling" as a real consumer of the module, having to consume the public APIs as a downstream user.